test/unit/AppLangSelectControllerSpec.js

```diff
@@ -0,0 +1,52 @@
+describe('AppLangSelectController', function () {
+  var $controller, $scope;
+
+  beforeEach(module('ui.bootstrap'));
+  beforeEach(module('izhukov.utils'));
+  beforeEach(module('myApp.i18n'));
+  beforeEach(module('myApp.services'));
+  beforeEach(module('myApp.controllers'));
+
+  beforeEach(function () {
```

Comment:
My first guess were global variables (hence I opened #1300). Will look closer at it.